src/java.base/share/classes/java/util/concurrent/ArrayBlockingQueue.java

```diff
@@ -509,7 +512,8 @@ public int remainingCapacity() {
      * @param o element to be removed from this queue, if present
      * @return {@code true} if this queue changed as a result of the call
      */
-    public boolean remove(@Nullable Object o) {
+    @CFComment("probably accepts null in practice, but docs at best imply this")
```

Comment:
It looks like https://github.com/typetools/checker-framework/pull/3063 didn't touch this file. (That is, this annotation was there before that PR.) I'm not sure if that means that you hadn't looked at it at the time or that you found some reason to justify omitting `@Nullable` here.

Likewise for the other queues/deques in this PR.